config/locales/en.yml

```diff
@@ -4,6 +4,7 @@ en:
   field_criticality: Criticality
   field_description: Description
   field_controls: Controls
+  field_income_statement: Income Statement
```

Comment:
I believe the specification dictated this should be `Quantification (Income Statement)`